engines/bops_api/app/jobs/bops_api/create_neighbour_boundary_geojson_job.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module BopsApi
+  class CreateNeighbourBoundaryGeojsonJob < ApplicationJob
+    queue_as :low_priority
+    discard_on ActiveJob::DeserializationError
+
+    def perform(planning_application)
+      response = ::Apis::OsPlaces::Query.new.find_addresses_by_radius(planning_application.lonlat.latitude, planning_application.lonlat.longitude).to_json
```

Comment:
Not sure we need top be converting this `to_json` so we can avoid doing the double parsing `JSON.parse(JSON.parse(response)["body"])["results"]`